js/index.js

```diff
@@ -0,0 +1,46 @@
+$(document).ready(function() {
+
+	//Set up variable to capture user input
+
+	var city= "none";
+
+	//Determine when user hits submit button
+
+	$('#myForm').submit(changeBackground);
+
+	function changeBackground(event){
+
+		//Stop page from reloading on submit
+		event.preventDefault();
+
+		//Get user input
+		var city = $('#city-type').val();
+
+
+		/*Match user input to determined keywords for cities and change 
+		background image to the specified city */
+		if(city==='New York'||city==='New York City'||city==='NYC'){
+		$('body').css('background', 'url(images/nyc.jpg)');
+		} else if(city==='SF' || city==="San Fransisco" || city==="Bay Area"||city==="san fransisco"){
```

Comment:
What about a lower case option for the other cities? Perhaps you could call this method when you store the variable - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase